renderer/index.js

```diff
@@ -318,7 +320,7 @@ function saveTorrentData (torrent) {
 function startTorrenting (torrentId) {
   var torrent = state.client.add(torrentId, {
     // use downloads folder
-    path: state.saved.downloads
+    path: state.saved.downloadPath
```

Comment:
#73 might be because of a mismatch between `state.saved.downloadPath` and `torrent.path`